skins/laika/src/view_models/AddressData.ts

```diff
@@ -0,0 +1,39 @@
+export interface PrivateDonorName {
+    salutation: string,
+    firstName: string,
+    lastName: string
+}
+
+export interface CompanyDonorName {
+    company: string
+}
+
+export interface DonorAddress {
+    streetAddress: string,
+    postalCode: string,
+    city: string,
+}
+
+export interface DonorEmail {
+    email: string
+}
+
+export interface PrivateDonor {
+    name: PrivateDonorName,
+    address: DonorAddress,
+    email: DonorEmail
+}
+
+export interface CompanyDonor {
+    name: CompanyDonorName,
+    address: DonorAddress,
+    email: DonorEmail
+}
+
```

Comment:
That's a lot of interfaces :smile: 
How about `PrivateDonor` and `CompanyDonor` merge into just `Donor` and the `name` would be `PrivateDonorName | CompanyDonorName`